### PR TITLE
[tools](tpc)make tpch-tools and tpcds-tools default scale factor 100

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

File: tools/tpch-tools/conf/doris-cluster.conf [hunk 1]
```diff
@@ -26,4 +26,4 @@ export USER='root'
 # Doris password
 export PASSWORD=''
 # The database where TPC-H tables located
-export DB='tpch_test'
+export DB='tpch_test2'
```
Comment (xzj7019) [inline tools/tpch-tools/conf/doris-cluster.conf:29]: invalid change? or just change it to tpch without _test following other benchmark tools.

Comment (englefly) [reply tools/tpch-tools/conf/doris-cluster.conf:29]: this is not proper. I will remove it

**Comment by englefly** [comment]
run buildall

**Review by Kikyou1997 — APPROVED** [review]

**Review by jackwener — APPROVED** [review]

**Pull request merged**